releasenotes/notes/add-eigensolvers-with-primitives-8b3a9f55f5fd285f.yaml

```
@@ -0,0 +1,12 @@
+---
+features:
+  - |
+     Added :class:`qiskit.algorithms.eigensolvers` package to include
+     interfaces for primitive-enabled algorithms. Refactored
+     :class:`qiskit.algorithms.eigensolvers.VQD` to leverage primitives.
+deprecations:
+  - |
+     :class:`qiskit.algorithms.eigen_solvers` package will now issue a ``PendingDeprecationWarning``. It
```

```suggestion
     :mod:`qiskit.algorithms.eigen_solvers` package will now issue a ``PendingDeprecationWarning``. It
```
This is a package/module not class